src/pages/join.tsx

```diff
@@ -24,9 +24,9 @@ const join: React.FC<{}> = () => (
     <body>
         <PageLinks />
         <BoardApps 
-            title="2021 Board Applications - Open through January 20th, 2021"
+            title="2021 Board Applications now Closed"
```

Comment:
capitalization is awk here I think, maybe just have "2021 Board Applications closed on January 20th, 2021" so ppl still have an approx idea for next year you know

Reply:
Fair